frontend/cypress.config.js

```diff
@@ -11,14 +12,13 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-import { defineConfig } from "cypress";
+import { defineConfig } from 'cypress';
 
 export default defineConfig({
-  e2e: {
-    setupNodeEvents(on, config) {
-      // implement node event listeners here
-    },
-    baseUrl: 'http://localhost:8080'
-
-  },
-});
\ No newline at end of file
+	e2e: {
+		setupNodeEvents(on, config) {
+			// implement node event listeners here
+		},
+		baseUrl: 'http://localhost:8080'
+	}
```

Comment:
Looks like tabs are used for indentation instead of spaces in these and the other places. I think spaces are preferred for consistent rendering across editors; the current code seems using spaces as well.